pkg/R/sparkRBackend.R

```diff
@@ -0,0 +1,85 @@
+# Methods to call into SparkRBackend. 
+
+# Call a Java method named methodName on the object
+# specified by objId. objId should be a "jobj" returned
+# from the SparkRBackend.
+callJMethod <- function(objId, methodName, ...) {
+  stopifnot(class(objId) == "jobj")
+  invokeJava(isStatic=FALSE, objId, methodName, ...)
+}
+
+# Call a static method on a specified className
+callJStatic <- function(className, methodName, ...) {
+  invokeJava(isStatic=TRUE, className, methodName, ...)
+}
+
+# Create a new object of the specified class name
+newJObject <- function(className, ...) {
+  invokeJava(isStatic=TRUE, className, methodName="new", ...)
```

Comment:
change the method name "new" to "<init>" (constructor name in byte code) in case there may be a method called "new" in a class?